.github/ISSUE_TEMPLATE/submission.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Proposal submission
+about: Propose a session for Our Networks 2019.
+title: '[Submission Title]'
+labels: ''
+assignees: ''
+---
+
+## Description
+
+[Description that is 1-2 paragraphs (75 - 150 words)]
+
+Type: [talk | panel | workshop | lightening talk | demos | performance | other (describe!)]  
+Length: [<30 minutes | 1 hour | 2 hour | half-day | N/A]  
+Additional considerations: [e.g., max number of participants, access]  
+
+## Session Objective
+
+[Identify 1-3 key objectives for the session (could be participant takeaways)]
+
+## Material and Technical Requirements
+
+[List materials you as presenter require AND any participant materials (indicate if there would be support required for a purchase/cost). Include equipment, technical, and installation requirements if applicable. We will aim to accommodate where possible]
+
+## Presenter(s)
+
+Name: Your Name  
+Email: name@youremail.com  
+Url(s): [yourwebsite.com](url to website), [yourotherwebsite.com](url to another website)  
+Twitter: [@you](url to twitter account)  
+GitHub: [you](url to GitHub account)  
+
+Interested in a community billet: [host|billet|N]
```

Comment:
```suggestion
Interested in a community billet: [host | billet | N]
```

Reply:
@garrying -- I'm having a hard time following this, do you want to have this go to my suggestion above to clarify billets, or rather keep this as a brief line? 

I'd like us to change the description because we received feedback this part was confusing and not adequately explained

Reply:
@dcwalk The description is good; it should go to your suggestion. It's adding spacing around the pipe for the options

Reply:
Okay! I think it doesn't stack on my suggestions, gonna to a commit of both of 'em and make sure it all looks good!

Reply:
Okay, it mucked it up. Lesson learned: only on suggestion per line. I added another commit with my language and yr white spacing